internal/kibana/reconciler.go

```diff
@@ -228,10 +242,21 @@ func (clusterRequest *KibanaRequest) createOrUpdateKibanaDeployment(proxyConfig
 		return kverrors.Wrap(err, "Failed to get oauth-proxy image")
 	}
 
+	oauthConfig, err := getOAuthConfig(clusterRequest.client)
+	if err != nil {
+		return kverrors.Wrap(err, "Failed to get oauth config")
+	}
+
+	oauthTimeout := 24 * time.Hour
```

Comment:
The `24 * time.Hour` is used a few times in the production and testing code. Should we have a constant for it?